chainer/link.py

```diff
@@ -205,14 +266,36 @@ def add_persistent(self, name, value):
                 for the attribute name.
             value: Value to be registered.
 
+        Returns:
+            The registered value.
```

Comment:
Same as `add_param`. Why do you change the function to return the registered value?